packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -170,18 +170,18 @@ export const GraphNG: React.FC<GraphNGProps> = ({
       const scaleColor = getFieldSeriesColor(field, theme);
       const seriesColor = scaleColor.color;
 
-      if (customConfig.axisPlacement !== AxisPlacement.Hidden) {
-        // The builder will manage unique scaleKeys and combine where appropriate
-        builder.addScale({
-          scaleKey,
-          distribution: customConfig.scaleDistribution?.type,
-          log: customConfig.scaleDistribution?.log,
-          min: field.config.min,
-          max: field.config.max,
-          softMin: customConfig.axisSoftMin,
-          softMax: customConfig.axisSoftMax,
-        });
```

Comment:
@leeoniya are you sure about this? I assume we are delegating combining the scales to uPlot?

Reply:
discussed, and yes